gef.py

```diff
@@ -4307,22 +4307,23 @@ class GefThemeCommand(GenericCommand):
-        self.add_setting("context_title_message", "red bold", "Color of the title in context window")
-        self.add_setting("default_title_line", "green bold", "Default color of borders")
-        self.add_setting("default_title_message", "red bold", "Default color of title")
-        self.add_setting("xinfo_title_message", "blue bold", "Color of the title in xinfo window")
-        self.add_setting("disassemble_current_instruction", "bold red", "Color to use to highlight the current $pc when disassembling")
-        self.add_setting("dereference_string", "green", "Color of dereferenced string")
-        self.add_setting("dereference_code", "red", "Color of dereferenced code")
-        self.add_setting("dereference_base_address", "bold green", "Color of dereferenced address")
-        self.add_setting("dereference_register_value", "bold green" , "Color of dereferenced register")
-        self.add_setting("registers_register_name", "bold red", "Color of the register name in the register window")
+        self.add_setting("context_title_line", "gray", "Color of the borders in context window")
+        self.add_setting("context_title_message", "cyan", "Color of the title in context window")
+        self.add_setting("default_title_line", "gray", "Default color of borders")
+        self.add_setting("default_title_message", "cyan", "Default color of title")
+        self.add_setting("table_heading", "blue", "Color of the column headings to tables (e.g. vmmap)")
+        self.add_setting("disassemble_current_instruction", "green", "Color to use to highlight the current $pc when disassembling")
+        self.add_setting("dereference_string", "yellow", "Color of dereferenced string")
+        self.add_setting("dereference_code", "gray", "Color of dereferenced code")
+        self.add_setting("dereference_base_address", "cyan", "Color of dereferenced address")
+        self.add_setting("dereference_register_value", "bold blue" , "Color of dereferenced register")
+        self.add_setting("registers_register_name", "blue", "Color of the register name in the register window")
         self.add_setting("registers_value_changed", "bold red", "Color of the changed register in the register window")
         self.add_setting("address_stack", "pink", "Color to use when a stack address is found")
-        self.add_setting("address_heap", "yellow", "Color to use when a heap address is found")
+        self.add_setting("address_heap", "green", "Color to use when a heap address is found")
         self.add_setting("address_code", "red", "Color to use when a code address is found")
-        self.add_setting("source_current_line", "bold red", "Color to use for the current code line in the source window")
+        self.add_setting("source_current_line", "green", "Color to use for the current code line in the source window")
+        self.add_setting("xinfo_title_message", "blue bold", "Unused")
```

Comment:
I don't want to remove this in case someone sets this setting in their own file. But I think it would just say an error and proceed, so maybe we should?

Reply:
Yeah I think it's okay to remove this. We'll just get an issue if someone complaints.